hun.json

```diff
@@ -0,0 +1,13 @@
+{
+    "loadButtonLabel": "Kattintson ide\na Panoráma\nBetöltéséhez",
+    "loadingLabel": "Betöltés...",
+	"bylineLabel": "%s által",    
+
+    "noPanoramaError": "Nincs megadott panorámakép.",
+    "fileAccessError": "A fájl %s nem elérhető.",
+    "malformedURLError": "Valami hibás a panoráma URL-jével.",
```

Comment:
Valami probléma van a panoráma URL-jével.
vagy
Valami hibás a panoráma URL-jében.

Reply:
With this I completely agree. I'm new at this, so how do I accept the proposed changes, or edit the pull request myself?